### PR TITLE
Beacons

### PR DESCRIPTION
**Pull request by sleepyyapril**
# Description

No merge this time, quite literally just basic changes. Should solve all issues, as this is how I did it on my fork.
Changes carp rifts and nuke arming to use readable locations.

---

<details><summary><h1>Media</h1></summary>
<p>

Beacons example

</p>
</details>

---

# Changelog
nah

### CODE REVIEW

**Review by VMSolidus — APPROVED** [review]
Ye

File: Content.Server/Dragon/DragonRiftSystem.cs [hunk 1]
```diff
@@ -72,9 +72,8 @@ public override void Update(float frameTime)
                 comp.State = DragonRiftState.AlmostFinished;
                 Dirty(comp);
 
-                var location = xform.LocalPosition;
                 _announcer.SendAnnouncement(_announcer.GetAnnouncementId("CarpRift"), Filter.Broadcast(),
-                    "carp-rift-warning", colorOverride: Color.Red, localeArgs: ("location", location));
+                    "carp-rift-warning", colorOverride: Color.Red, localeArgs: ("location", FormattedMessage.RemoveMarkup(_navMap.GetNearestBeaconString((uid, xform)))));
```
Comment (Remuchi) [inline Content.Server/Dragon/DragonRiftSystem.cs:76]: Iirc `RemoveMarkup` is obsolete and should be changed to `RemoveMarkupPermissive`

File: Content.Server/Nuke/NukeSystem.cs [hunk 2]
```diff
@@ -467,7 +462,7 @@ public void ArmBomb(EntityUid uid, NukeComponent? component = null)
             Color.Red,
             stationUid ?? uid,
             null,
-            ("time", (int) component.RemainingTime), ("position", posText)
+            ("time", (int) component.RemainingTime), ("position", FormattedMessage.RemoveMarkup(_navMap.GetNearestBeaconString((uid, nukeXform))))
```
Comment (Remuchi) [inline Content.Server/Nuke/NukeSystem.cs:465]: same here

**Review by Remuchi — APPROVED** [review]
I'd really would like to have it changed before merging but whatever.

File: Content.Server/Dragon/DragonRiftSystem.cs (hunk 1 above)
Comment (sleepyyapril) [reply Content.Server/Dragon/DragonRiftSystem.cs:76]: Doesn't exist, I think. Ctrl+shift+f

Comment (sleepyyapril) [reply Content.Server/Dragon/DragonRiftSystem.cs]: Nevermind, was an RT thing. Done

**Pull request merged**